### DISCUSSION TITLE
cfclient not finding crazyradio 2.0 after changing bandwidth to 250 kbit/s

### DISCUSSION THREAD

**Discussion by jjdike** [bitcraze/discussions#1162] [Q&A] [answered]
Hi - the cfclient had been finding the crazyradio 2.0 dongle regularly but when I changed the bandwidth to 250 kbit/s, following the firmware configuration section of https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/ cfclient is no longer finding the dongle (with crazyflie turned on).   I can fly crazyflie with the phone app.  What should I try to get the dongle to be recognized again?

Thanks.

**Comment by knmcguire** [comment]
Hi!

So it is indeed the case that the Crazyradio 2.0 can no longer support 250 kbits/s. However, the dongle should still lbe recognized by the cfclient, so I think you perhaps mean that the dongle is not able to find the crazyflie anymore?  The mobile app uses bluetooth so it's not using the same protocol.

What you should do now is connect the crazyflie with an microusb cable to the computer and click scan so that it shows up as usb://0, and configure the Crazyflie back to 2mbit/sec again.

**Reply by jjdike** [reply]
Hi,

When I connect crazyflie by usb, cfclient does not see the usb://0 when scanning.  I do see it in 'other devices' in the windows device manager.  I just mentioned flying with the app to say that the crazyflie itself seemed to be working ok, with the normal start up sequence.

When I insert the crazyradio 2.0 dongle, it shows up under libusbK devices in the device manager.  However a scan from cfclient with crazyflie on or off does not lead to the crazyradio showing up in the cfclient interface.

**Reply by knmcguire** [reply, marked as answer]
Ah wait, if you are windows, you'll need to install the zadig driver for the Crazyflie as well, same way as what you had to do for the crazyradio 2.0

**Reply by jjdike** [reply]
Yes that allowed me to see the crazyflie from cfclient and adjust the radio rate to a value other than 250k/s.  Thanks. 

Unfortunately I'm still not able to see the radio dongle from cfclient.  I tried reflashing, and i can see the red led on the dongle light up when scanning, but it's not found as an interface.

This is what I see in the command window when scanning:

INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Found 0 devices
INFO:cflib.crtp.radiodriver:v3.2 dongle with serial N/A found
INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Found 0 devices
INFO:cflib.drivers.cfusb:Looking for devices....
INFO:cflib.crtp.prrtdriver:Initialized PRRT driver.
INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices


Just an update.  I closed cfclient, reinserted the radio dongle, and turned crazyflie on/off and rescanned - now the dongle is found.
